src/evse_man.h

```diff
@@ -449,6 +452,9 @@ class EvseManager : public MicroTasks::Task
     void enableTemperatureCheck(bool enabled, std::function<void(int ret)> callback = NULL) {
       _monitor.enableTemperatureCheck(enabled, callback);
     }
+    void restartEvse() {
```

Comment:
@matth-x I think there was an outstanding requirement for restarting the EVSE for OCPP, does this meet that requirement?